Gopkg.lock

```diff
@@ -467,6 +475,7 @@
   analyzer-version = 1
   input-imports = [
     "github.com/DATA-DOG/go-sqlmock",
+    "github.com/NYTimes/gziphandler",
```

Comment:
what's this used for? I don't see it in the code diff.

Reply:
Looks like an accident... I agree, I wouldn't have expected `Gopkg.lock` changes here.

Lockfile generally only changes if you've decided to update dependencies (which  probably belongs in its own PR) or because you're changing `Gopkg.toml`.

Reply:
@mkendall07  @dbemiller. This is used in `server.go` and is a result of I running `dep ensure` on my machine to make sure I have all the updated dependencies. Wasn't aware of the fact that dependencies are suppose to be updated as part of a dedicated PR. I will reverse this. Thanks for pointing it out.

Reply:
oh man.... that's a quirk in `dep` that I never knew about o.o.

Apparently it'll just auto-fetch anything you `import`, regardless of whether or not you've put it in your `Gopkg.toml`. I was treating it like `npm`, which I'm more familiar with, and assumed you had to list the dependency explicitly to pull it down.

That's... quite dangerous, because it means the builds aren't reproducible now.  I take it back... you definitely want to get that in your lockfile as soon as possible :). It shouldn't _update_ dependencies... it'll just pin them to a known version.

Good thing [native go modules](https://github.com/golang/go/wiki/Modules) are in the process of replacing `dep`, eh?

Reply:
Yes, `dep ensure` will usually fetch something that you've imported but not specified in the `Gopkg.toml` file. I think the thought process with `dep` here is that if your code is using it, `dep` will be like, _I see you're importing this dependency and using it. I will just install it for you. And I will abide by any rules for installing it if you've mentioned them in the `Gopkg.toml` file._ 

And for reproducible builds, I believe you'd have to commit and track `vendor` as part of the github repo too.

As for the `gziphandler`, we do have that in our `Gopkg.toml` file. I am now confused why didn't it exist in the lock file until now though 😅

Reply:
very odd...

akh...  sounds like a sloppy code review on our part. Whichever PR updated the `Gopkg.toml` must not have also  updated the `Gopkg.lock` like it should have. Probably my bad.

> And for reproducible builds, I believe you'd have to commit and track vendor as part of the github repo too.

technically true, yeah :). Should have said "more reproducible" builds. Without the lockfile  your  build won't reproduce  if they, say, put out a new version of their library.  Or  if they don't have tagged versions yet, put out any new commits at all.

Reply:
Yes. You definitely need the `Gopkg.lock` file. It is what defines the current state of all your imports. Just need to always be mindful that a `dep ensure` won't always lead to an exact same build 😅 

Anyway, I have kept the `Gopkg.lock` file changes in this PR.